.scons_env.json

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
This file should not be added, it's ignored by `.gitignore`. You need to rebase on the current `master` branch and remove it.